### PR TITLE
CaaSP system locking

### PR DESCRIPTION
**Pull request by mateiw**
Update docs with info about system locking when CaaSP is detected.

### CODE REVIEW

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 1]
```diff
@@ -20,11 +20,11 @@ If you are using ``cloud-init``, we recommended that you use a bootstrap script
 For more on bootstrapping, see xref:client-configuration:registration-bootstrap.adoc[].
 
 
-When you have added the {caasp} nodes to {productname}, you will be able to:
+When you have added the {caasp} nodes to {productname}, the registered system will be locked automatically.
+A warning will be shown in the UI that the system is locked. The UI and API will allow the user to schedule any action but this action will fail as long as the system is locked.
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc]:
```suggestion
When the system is locked, the {webui} shows a warning. 
When this happens, you can schedule actions using the {webui} or the API, but the action will fail.
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 2]
```diff
@@ -20,11 +20,11 @@ If you are using ``cloud-init``, we recommended that you use a bootstrap script
...
 
-* View the patch level status of each node.
-* Perform configuration management operations against the nodes, including using Salt states.
-* Assign different set of channels to clusters using channel staging and lifecycle management filters.
+The system lock can be enabled or disabled from the System Lock formula. When the system lock is disabled, all operations are permitted.
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc]:
```suggestion
You can enable or disable the system lock using the System Lock formula. 
When the system lock is disabled, all operations are permitted. 
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 3]
```diff
@@ -20,11 +20,11 @@ If you are using ``cloud-init``, we recommended that you use a bootstrap script
...
 
-When you have added the {caasp} nodes to {productname}, you will be able to:
+When you have added the {caasp} nodes to {productname}, the registered system will be locked automatically.
+A warning will be shown in the UI that the system is locked. The UI and API will allow the user to schedule any action but this action will fail as long as the system is locked.
 
-* View the patch level status of each node.
-* Perform configuration management operations against the nodes, including using Salt states.
-* Assign different set of channels to clusters using channel staging and lifecycle management filters.
+The system lock can be enabled or disabled from the System Lock formula. When the system lock is disabled, all operations are permitted. 
+This System Lock formula is enabled automatically if {caasp} is detected on the node.
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc]:
```suggestion
The System Lock formula is enabled automatically if {caasp} is detected on the node.
```

**Comment by Loquacity** [comment]
@mateiw can you please approve the suggestions and merge? I can't do it on your fork.

**Comment by mbologna** [comment]
@Loquacity @keichwa please review again. After your OK I'll ask @mateiw to merge. Thanks

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 4]
```diff
@@ -19,12 +17,17 @@ For more on activation keys, see xref:client-configuration:clients-and-activatio
 If you are using ``cloud-init``, we recommended that you use a bootstrap script in the ``cloud-init`` configuration.
 For more on bootstrapping, see xref:client-configuration:registration-bootstrap.adoc[].
 
+When you have added the {caasp} nodes to {productname}, the registered system will be locked automatically.
+When a system is locked, the {webui} shows a warning and you can schedule actions using the {webui} or the API, but the action will fail.
+
+[NOTE]
+The locking mechanism works only with plain Salt minions (Locking is not supported with salt-ssh minions).
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc]:
```suggestion
====
The locking mechanism works only with plain Salt minions (Locking is not supported with salt-ssh minions).
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 5]
```diff
@@ -19,12 +17,17 @@ For more on activation keys, see xref:client-configuration:clients-and-activatio
...
 
-When you have added the {caasp} nodes to {productname}, you will be able to:
+You can enable or disable the system lock using the System Lock formula.
+When the system lock is disabled, all operations are permitted.
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc]:
```suggestion
When the system lock is disabled, all operations are permitted.
====
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 6]
```diff
@@ -19,12 +17,17 @@ For more on activation keys, see xref:client-configuration:clients-and-activatio
...
 
-* View the patch level status of each node.
-* Perform configuration management operations against the nodes, including using Salt states.
-* Assign different set of channels to clusters using channel staging and lifecycle management filters.
+[NOTE]
+The System Lock formula is enabled automatically if {caasp} is detected on the node.
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc]:
```suggestion
====
The System Lock formula is enabled automatically if {caasp} is detected on the node.
====
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 7]
```diff
@@ -42,6 +45,6 @@ When using Salt or {productname} (either via UI or API) on any {caasp} nodes:
 * Do not remove a package if it breaks or conflicts or is one of the packages related with the `patterns-caasp-Node-x.y`
 * Do not upgrade a package if it breaks or conflicts or is one of the packages related with the `patterns-caasp-Node-x.y`
 
-{productname} will not stop you from issuing these operations.
-Issuing those operation may render your {caasp} cluster unusable.
+Issuing those operations may render your {caasp} cluster unusable.
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc]:
```suggestion
Issuing those operations could render your {caasp} cluster unusable.
```

**Comment by keichwa** [comment]
I'll recheck when Lana's suggestions are in (if still needed).

**Pull request merged**